### PR TITLE
Saml

### PR DESCRIPTION
**Pull request by z4kn4fein**
### Related tickets

[Ticket](https://trello.com/c/Ki1o7NEr/125-docs)

### Description

This PR adds the IdP configuration tutorials and other docs related to SAML.

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

File: website/docs/advanced/user-management/saml/overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+id: sso-overview
+title: Overview
+---
+
+This section describes how you can enable SAML Single Sign-On (SSO) for your ConfigCat environment.
+
+SAML SSO allows your team members to authenticate with ConfigCat via their company accounts using your own Identity Provider (IdP).
```
Comment (sigewuzhere) [inline website/docs/advanced/user-management/saml/overview.md:8]: In the app I called this `SAML authentication` not `SAML SSO`. Any of them wins, it should be the same everywhere.

File: website/sidebars.js [hunk 2]
```diff
@@ -13,10 +13,31 @@ module.exports = {
       "advanced/user-object",
       "advanced/public-api",
       "advanced/caching",
-      "advanced/team-collaboration",
+      {
+        "User Management": [
```
Comment (sigewuzhere) [inline website/sidebars.js:17]: Should be `Team Management` as in the Dashboard

File: website/sidebars.js [hunk 3]
```diff
@@ -13,10 +13,31 @@ module.exports = {
...
+          "advanced/user-management/team-collaboration",
+          {
+            "SAML Single Sign-On": [
+              "advanced/user-management/saml/sso-overview",
```
Comment (sigewuzhere) [inline website/sidebars.js:21]: routes should reflect on the final naming of the feature

File: website/docs/advanced/user-management/team-collaboration.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 ---
 id: team-collaboration
-title: Team and Collaboration Management
+title: Team and Collaboration
```
Comment (sigewuzhere) [inline website/docs/advanced/user-management/team-collaboration.md:3]: Team Management Basics

File: website/sidebars.js (hunk 2 above)
Comment (sigewuzhere) [reply website/sidebars.js:17]: Sidebar item ordering by importance:
- Team Management
  - Team Management Basics
  - Auto-assign users
  - SAML
  - Domain validation

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

File: website/docs/advanced/user-management/saml/overview.md (hunk 1 above)
Comment (z4kn4fein) [reply website/docs/advanced/user-management/saml/overview.md]: ✔️

File: website/sidebars.js (hunk 2 above)
Comment (z4kn4fein) [reply website/sidebars.js]: ✔️

File: website/sidebars.js (hunk 3 above)
Comment (z4kn4fein) [reply website/sidebars.js]: ✔️

File: website/docs/advanced/user-management/team-collaboration.md (hunk 4 above)
Comment (z4kn4fein) [reply website/docs/advanced/user-management/team-collaboration.md]: ✔️

File: website/docs/advanced/team-management/saml/overview.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- [OneLogin](identity-providers/onelogin)
+
+:::info
+Other Identity Providers might also work with ConfigCat, if they support the SAML 2.0 protocol.
+:::
+
+
+### Supported SAML flows
+  - IdP initiated SSO
+  - SP initiated SSO
```
Comment (laliconfigcat) [inline website/docs/advanced/team-management/saml/overview.md:32]: Itt én lehet kiírnám, hogy (Identitiy provider) meg hogy mi is az az SP. Én így hirtelen nem is tudom, service provider?

File: website/docs/advanced/team-management/auto-assign-users.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+id: auto-assign-users
+title: Auto-assign Users
+---
+
+All new users who sign-up with a [verified email domain](./domain-verification) will be automatically added to your organization.
```
Comment (laliconfigcat) [inline website/docs/advanced/team-management/auto-assign-users.md:6]:
```suggestion
All new users who sign up with a [verified email domain](./domain-verification) will be automatically added to your organization.
```

File: website/docs/advanced/team-management/saml/overview.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+id: saml-overview
+title: Overview
+---
+
+This section describes how you can enable SAML Single Sign-On (SSO) for your ConfigCat environment.
+
+SAML SSO allows your team members to authenticate with ConfigCat via their company accounts using your own Identity Provider (IdP).
```
Comment (laliconfigcat) [inline website/docs/advanced/team-management/saml/overview.md:8]:
```suggestion
SAML SSO allows your team members to sign up and log in to ConfigCat via their company accounts using your own Identity Provider (IdP).
```

File: website/docs/advanced/team-management/saml/identity-providers/auth0.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+### 2. Configure SAML for the Okta Application
+- Open your organization's authentication settings on the <a href="https://app.configcat.com/organization/authentication" target="_blank">ConfigCat dashboard</a>.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/dashboard/authentication.png" />
+
+- Select the domain you want to configure with SAML, and click `Set` under the `SAML SSO status`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/dashboard/domains.png" />
+
+- From the appearing dialog, copy the following values and paste them into the configuration dialog.
```
Comment (laliconfigcat) [inline website/docs/advanced/team-management/saml/identity-providers/auth0.md:39]:
```suggestion
- From the appearing dialog, copy the following values and paste them into the Auth0 configuration dialog.
```

File: website/docs/advanced/team-management/saml/identity-providers/onelogin.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+- Log in to <a href="https://app.onelogin.com/login" target="_blank">OneLogin</a>, and select `Applications`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/onelogin/applications.png" />
+
+- Click on `Add App`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/onelogin/add_app.png" />
+
+- Type `saml` into the search bar, and select `SAML Custom Connector (Advanced)`.
```
Comment (laliconfigcat) [inline website/docs/advanced/team-management/saml/identity-providers/onelogin.md:24]:
```suggestion
- Type `SAML` into the search bar, and select `SAML Custom Connector (Advanced)`.
```

File: website/docs/advanced/team-management/saml/identity-providers/onelogin.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+  <img class="saml-tutorial-img" src="/static/assets/saml/dashboard/authentication.png" />
+
+- Select the domain you want to configure with SAML, and click `Set` under the `SAML SSO status`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/dashboard/domains.png" />
+
+- From the appearing dialog, copy the following values and paste them into the OneLogin application's configuration page.
+    - Copy `Entity ID` and paste it into the `Audience (EntityID)`.
+    - Copy `Assertion Consumer Service` and paste it into the `ACS (Consumer) URL`.
+    - Paste the same `Assertion Consumer Service` into the `ACS (Consumer) URL Validator` in regex format.
```
Comment (laliconfigcat) [inline website/docs/advanced/team-management/saml/identity-providers/onelogin.md:46]: Fú, ez elég combos. Vajon nem tudnánk valahogy segíteni?

**Review by laliconfigcat — CHANGES_REQUESTED** [review]

File: website/docs/advanced/team-management/domain-verification.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
+---
+id: domain-verification
+title: Domain Verification
+---
+#### Steps to verify your domain:
```
[the remark this answers is not in the document]
Comment (z4kn4fein) [reply website/docs/advanced/team-management/domain-verification.md:8]: Done

File: website/docs/advanced/team-management/saml/overview.md (hunk 5 above)
Comment (z4kn4fein) [reply website/docs/advanced/team-management/saml/overview.md]: Done

File: website/docs/advanced/team-management/saml/identity-providers/onelogin.md (hunk 10 above)
Comment (z4kn4fein) [reply website/docs/advanced/team-management/saml/identity-providers/onelogin.md:46]: Hat, nekem is ki kellett szopkodnom az eterbol, nem igazan van jo leiras errol, de a screenshoton ott a pelda legalabb, csak a vegen a guid-ot kell lecserelni

**Review by laliconfigcat — DISMISSED** [review]
You are the hero!!!
hero-cat-only (2)

**Review by laliconfigcat — APPROVED** [review]

**Pull request merged**